packages/blocks/src/root-block/widgets/doc-remote-selection/config.ts

```diff
@@ -0,0 +1,5 @@
+import type { BlockModel } from '@blocksuite/store';
+
+export type DocRemoteSelectionConfig = {
+  blockSelectionBackgroundTransparent: (block: BlockModel) => boolean;
```

Comment:
Maybe we can make this transparent selection background a default feature for block selections?

Reply:
@Svaney-ssman say text-related blocks can keep the previous style, like `paragraph`, `list`. If use transparent background by default. The style of following blocks selections may be wired

![image](https://github.com/user-attachments/assets/f04e6db8-8b5d-4550-bbbc-6078a689e933)

Reply:
But I think we can transparent all no text related block background by default, and remove the config interface. What do you think?